cadence/contracts/FCLCrypto.cdc

```diff
@@ -0,0 +1,108 @@
+pub contract FCLCrypto {
```

Comment:
So does the dev wallet assume that `FCLCrypto` is already deployed to the emulator?

Just thinking about how make sure `FCLCrypto` is available to a developer on the emulator. The dev wallet _could_ deploy it (like it does for the `FCL` contract), but I'm not sure that's the best pattern.

Maybe it should become one of the default contracts added to the emulator on startup? cc @10thfloor

Reply:
Wallet doesn't assume (as it doesn't verify) but dapp configured to use emulator will require an address where contract is deployed. I think adding to default contracts on startup is the move